.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+
+## Related issues
+
+* none
+
+## Related code and/or documentation pull requests
+
+* none
+
+## Documentation
+
+- [ ] User facing documentation not needed
```

Comment:
Can we remove `docs-needed` label now?

Reply:
Not sure yet .. we are just assembling more info in the PR description for now .. labels can help us as well. We are figuring this out. Labels are not consistently used at this stage